front/app/containers/Admin/dashboard/reports/ProjectReport.tsx

```diff
@@ -312,33 +317,61 @@ const ProjectReport = memo(
                 />
               </>
             </GraphsContainer>
-          </Section>
-        )}
-      </>
+          )}
+          {participationMethods.includes('poll') ? (
+            isTimelineProject ? (
+              !isNilOrError(phases) &&
+              phases.map(
+                (phase) =>
+                  phase.attributes.participation_method === 'poll' && (
+                    <PollReport
+                      participationContextType="phase"
+                      participationContextId={phase.id}
+                      participationContextTitle={localize(
+                        phase.attributes.title_multiloc
+                      )}
+                    />
+                  )
+              )
+            ) : (
+              <PollReport
+                participationContextType="project"
+                participationContextId={project.id}
+                participationContextTitle={localize(
+                  project.attributes.title_multiloc
+                )}
+              />
+            )
+          ) : null}
+        </Section>
+      </ThemeProvider>
     );
   }
 );
 
-const ProjectReportWithHoc = injectIntl(ProjectReport);
+const ProjectReportWithHoc = withRouter(injectIntl(ProjectReport));
```

Comment:
The WithRouterProps are used outside of this component in the Data component, so it should be wrapped around the export. See comment below.